### PR TITLE
Improve config setup

### LINKED ISSUE #14: Easier way to switch to different AWS accounts
Currently, user has to override the `setupDynamoDb()` method.

### PR DESCRIPTION
**Pull request by baopham**
Closes #14

---
This change is [Reviewable](https://reviewable.io/reviews/baopham/laravel-dynamodb/97)

### CODE REVIEW

**Comment by baopham** [comment]
@zoul0813 could you give this branch a try?

File: src/DynamoDbClientService.php [hunk 1]
```diff
@@ -22,19 +23,33 @@ class DynamoDbClientService implements DynamoDbClientInterface
...
     {
-        return $this->client;
+        $connection = $connection ?: config('dynamodb.default');
+
+        if (isset($this->clients[$connection])) {
+            return $this->clients[$connection];
+        }
+
+        $config = config("dynamodb.connections.$connection", []);
+        $config['version'] = '2012-08-10';
```
Comment (zoul0813) [inline src/DynamoDbClientService.php]: Should the version be hard-coded here ... or should we allow the user to set it in the config, and only set it here if the user omits it?

File: src/DynamoDbModel.php [hunk 2]
```diff
@@ -12,6 +12,13 @@
 abstract class DynamoDbModel extends Model
 {
     /**
+     * The connection name for the model.
+     *
+     * @var string
+     */
+    protected $connection;
```
Comment (zoul0813) [inline src/DynamoDbModel.php]: This variable exists on the Eloquent Model already - are you just defining it here for clarity?

File: src/DynamoDbClientService.php (hunk 1 above)
Comment (baopham) [reply src/DynamoDbClientService.php]: I thought about it too 🤔  but how our library is implemented quite depends on the version of the AWS API. This is like indicating our dependency.

File: src/DynamoDbModel.php (hunk 2 above)
Comment (baopham) [reply src/DynamoDbModel.php]: At the time of writing, yeah it was the intention. But then, we are not listing all of the properties here anyway (`primaryKey`, `table`, etc.). So I'll remove it.

**Comment by zoul0813** [comment]
All in all - looks good.  Tests ran successfully on my end.

**Comment by baopham** [comment]
Awesome, thanks! will be cleaning up based on the comments and update README.

File: src/DynamoDbClientService.php (hunk 1 above)
Comment (zoul0813) [reply src/DynamoDbClientService.php]: Ok, I can see that as a valid reason.

**Pull request merged**